Dockerfile

```diff
@@ -12,17 +12,13 @@
 # the License.
 #
 
-apiVersion: v1
-kind: ImageStream
-metadata:
-  name: jaeger-agent
-spec:
-  lookupPolicy:
-    local: false
-  tags:
-    - from:
-        kind: DockerImage
-        name: ${jaeger.agent.image}
-      name: latest
-      referencePolicy:
-        type: Source
+FROM openjdk:alpine
+
+RUN apk --update --no-cache add tar && rm -rf /var/cache/apk/*
```

Comment:
why is this needed?

Reply:
I was using tar command earlier. I do remove this.